src/dr_llm/llm/providers/anthropic/request.py

```
@@ -51,9 +51,8 @@ def from_llm_request(
         request: LlmRequest,
         config: AnthropicConfig,
     ) -> AnthropicRequest:
-        if request.max_tokens is None:
-            if request.provider != "minimax":
-                raise ProviderSemanticError("anthropic requests require max_tokens")
+        if request.max_tokens is None and request.provider != "minimax":
+            raise ProviderSemanticError("anthropic requests require max_tokens")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)